apps/examples/e2e/tests/fixtures/menus/toolbar.ts

```diff
@@ -0,0 +1,37 @@
+import { Locator, Page, expect } from '@playwright/test'
+
+export class Toolbar {
+	readonly toolLock: Locator
+	readonly moreToolsButton: Locator
+	readonly moreToolsPopover: Locator
+	readonly mobileStylesButton: Locator
+	readonly tools: { [key: string]: Locator }
+	readonly popoverToolNames: { [key: string]: Locator }
+
+	constructor(public readonly page: Page) {
+		this.page = page
+		this.toolLock = this.page.locator('[data-testid="tool-lock"]')
+		this.moreToolsButton = this.page.locator('[data-testid="tools.more-button"]')
+		this.moreToolsPopover = this.page.locator('[data-testid="tools.more-content"]')
+		this.mobileStylesButton = this.page.locator('[data-testid="mobile-styles.button"]')
+		this.tools = {
+			select: this.page.locator('[data-testid="tools.select"]'),
+			draw: this.page.locator('[data-testid="tools.draw"]'),
+			arrow: this.page.locator('[data-testid="tools.arrow"]'),
+			cloud: this.page.locator('[data-testid="tools.cloud"]'),
+			eraser: this.page.locator('[data-testid="tools.eraser"]'),
+		}
+		this.popoverToolNames = {
```

Comment:
Could maybe also rename this to `popoverTools`, just as we did with `toolNames` -> `tools`.